daemon/entrypoint.sh

```diff
@@ -65,6 +65,11 @@ chown ceph. /var/run/ceph
 # based on the data store
 case "$KV_TYPE" in
    etcd|consul)
+      if [[ ! -e "/etc/ceph/populate.sh" ]]; then
```

Comment:
This seems like a strange way to phrase this.  I would prefer a direct check for the file to be sourced.  If the intent is to allow an override of the populate script, it should be:
  - commented as such in the script
  - phrased directly (if file exists, versus if file does _not_ exist)
  - documented in the associated README